<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssetPickerWidget</class>
 <widget class="QWidget" name="AssetPickerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
    background-color: #4D4D4D;
    color: #DEDEDE;
    border: 0;
	padding: 2px 8px;
}

QPushButton:hover {
    background-color: #555;
}

QPushButton:pressed {
    background-color: #444;
}

QSlider::groove:horizontal {
   /* height: 8px; /* the groove expands to the size of the slider by default. by giving it a height, it has a fixed size */
    background: #404040;
}

QSlider::handle:horizontal {
    background: #AFAFAF;
    width: 10px;
    margin: -2px 0; /* handle is placed by default on the contents rect of the groove. Expand outside the groove */
}

QListView {
	background: #303030;
	border: 0;
}

QListWidget {
	background: #404040;
	color: #CECECE;
}

QListView::item:selected {
    border: 1px solid #3498db;
	background: #3498db;
	color: #CECECE;
}

/* ================== */

 QScrollBar:vertical {
     background-color: #212121;
     width: 18px;
     margin: 22px 0 22px 0;
 }

 QScrollBar::handle:vertical {
	background-color: #444;
     min-height: 20px;
     margin-left: 2px;
     margin-right: 2px;
 }

QScrollBar::handle:vertical:hover {
	background-color: #555;
	min-height: 20px;
	margin-left: 2px;
	margin-right: 2px;
}

QScrollBar::add-line:vertical {
	background-color: #444;
	height: 20px;
	border: 2px solid #212121;
	border-width: 2px 2px  0 2px;
	subcontrol-position: bottom;
	subcontrol-origin: margin;
 }

QScrollBar::sub-line:vertical {
    background-color: #444;
	height: 20px;
	border: 2px solid #212121;
	border-width: 0 2px 2px 2px;
	subcontrol-position: top;
	subcontrol-origin: margin;
 }

QScrollBar::up-arrow:vertical {
    image: url(:/app/icons/up-arrow.svg);
}

QScrollBar::up-arrow:vertical:hover {
    background-color: #555;
}

QScrollBar::down-arrow:vertical {
    image: url(:/app/icons/down-arrow.svg);
}

QScrollBar::down-arrow:vertical:hover {
    background-color: #555;
}

/* ========================== */

/* For icon only */
/*QListView::icon {
    left: 10px;
}*/

/* For text only */
/*QListView::text {
    left: 10px;
}*/</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="searchBar"/>
     </item>
     <item>
      <widget class="QPushButton" name="viewButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/app/icons/view-list-button.svg</normalon>
         <activeon>:/app/icons/list-button-with-3-elements.svg</activeon>
        </iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="scanBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Refresh</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QListWidget" name="assetView">
     <property name="styleSheet">
      <string notr="true">QScrollBar:vertical {
	background: #1e1e1e;
}</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
